eslint/src/rules/4/deprecated.media-rule-list-parse.ts

```diff
@@ -8,11 +8,12 @@ import type {ESLintReplacementMethodCfg} from '../../core/deprecated-class-metho
 
 const AVAILABLE_SINCE = '5.0.0-beta.24';
 const isActual = lte(ESL_PACKAGE_VERSION, AVAILABLE_SINCE);
+const isTest = process.env.NODE_ENV === 'test';
 
 /**
  * Rule for deprecated 'parse' method of {@link ESLMediaRuleList}
  */
-export default isActual ?
+export default isActual || isTest ?
```

Comment:
We probably can just disable that rule or exclude it from defalat at that point.

Reply:
> We probably can just disable that rule or exclude it from defalat at that point.

If we exclude it, do we to remove tests too?

Reply:
I'd say no, tests should be isolated and should not affect anything